mcaleb808/myPharmBackend
# View all membership requests

---

#### Description of changes made

> Admin can view all the membership requests.

#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/mcaleb808/myPharmBackend.git), and `cd into the project`
- Check out this branch by _git checkout `branchName`_
- Run `yarn | npm install` to install the project dependencies
- Make sure to have your `.env` file set up correctly
- Run `yarn dev | npm run dev` to start the local server.
- using `Postman` send `GET` request to `/api/v1/requests` to view all requests.

#### Checklist:


#### GitHub issue your changes fix

Resolves #17

#### Screenshots
N/A
